news/2018-02-26-aspire-2017-stage-three-osucatch-results.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Aspire 2017 Stage Three: osu!catch Results"
+date: 2018-02-26 12:00:00 +0000
+---
+
+The fruits fell near-endlessly, and the world was a much healthier (and more interesting) place for them. Who would rise to the top in osu!catch's first official "anything-goes" mapping contest? Read on to find out!
+
+[![](https://assets.ppy.sh/contests/51/header.png)](https://osu.ppy.sh/community/contests/51)
+
+Faced with the task of mapping the appropriately food-themed _ONIGIRI FREEWAY_ by Featured Artist [OISHII](https://osu.ppy.sh/beatmaps/artists/17), fruit-arrangers came from far and wide to test their mapping might.
+
+Last month's live showcase featured ten of the most creative offerings seen in osu!catch mapping to date, savoured and scrutinised by our panel of expert judges, before allowing you, the community, to decide each mapper's fruity fate.
+
+When the dust settled, only one would top the mountain, but it would do so with an utterly dominant performance that blew all other competition out of the water. Receiving critical praise from the community and judging panel alike, the winner of Aspire 2017: Stage Three is perhaps fittingly, **Superior Fox**, mapped by the one and only [Spectator](https://osu.ppy.sh/users/702598), with a total score of **72.2/80** and **218** unique public votes!
+
+<iframe width="100%" height="720" src="https://www.youtube.com/embed/mJqI-W7R8gI&list=UUMeRgqzTfC5ja40B6kM6pdg" frameborder="0" allowfullscreen></iframe>
```

Comment:
This embed link failed for me. Omitting `&list=UUMeRgqzTfC5ja40B6kM6pdg` fixed it though.
(this happens multiple times)

Reply:
Recopied the embed code direct from the YT links instead of cannibalising taiko Aspire announcement